src/models/User.ts

```diff
@@ -0,0 +1,13 @@
+import * as mongoose from "mongoose";
+
+const UserSchema = new mongoose.Schema({
+  login: { type: String, required: true, unique: true },
+  email: { type: String, required: true, unique: true },
+  password: { type: String, required: true, unique: true },
+  registrationDate: Date,
+  lastLoggedIn: Date,
+});
+
+const UserModel = mongoose.model("User", UserSchema);
+
+export {UserModel};

```

Comment:
Add User Interface https://stackoverflow.com/questions/34482136/mongoose-the-typescript-way

Reply:
done